notification/src/api/notification/notification.controller.js

```diff
@@ -21,7 +21,8 @@ async function handleNotification (request, response) {
       { 'Content-Type': 'application/json' },
       JSON.stringify({ notificationResponse: '[accepted]' }))
   } catch (e) {
-    logger.error(e, 'Unexpected exception occurred')
+    logger.error(e,
+      `Unexpected exception occurred. Received payload from Adyen: ${body}`)
```

Comment:
It might be better to see body as JSON and not as an escaped JSON text. Did you try:
```
const customErr = { err, "adyenRequestBody": body}
logger.error({customErr}, "Unexpected exception occurred")
```

Reply:
We lose stacktrace in this case. It looks like we should implement a custom serializer to put it in a separate field. Didn't find an easy solution. Let's keep it as it is?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/53/commits/99aca3f00c474a8c251cf7ef2c63d1e20ce9b6d0